contrib/chart/backstage/templates/backend-secret.yaml

```diff
@@ -21,4 +21,6 @@ stringData:
   NEW_RELIC_REST_API_KEY: {{ .Values.auth.newRelicRestApiKey }}
   TRAVISCI_AUTH_TOKEN: {{ .Values.auth.travisciAuthToken }}
   PAGERDUTY_TOKEN: {{ .Values.auth.pagerdutyToken }}
+  SPLUNK_ON_CALL_API_ID: {{ .Values.auth.splunkOnCallApiId }}
+  SPLUNK_ON_CALL_API_KEY: {{ .Values.auth.splunkOnCallApiKey }}
```

Comment:
Again, would suggest we may not want to change this globally for this contrib sample, but if needed adding it to the README for the plugin setup instructions?